public/index.html

```diff
@@ -1,12 +1,11 @@
-<!doctype html>
-<html>
+<!DOCTYPE html>
+<html lang="en">
   <head>
     <meta charset="UTF-8">
     <title>Hello React</title>
-    <script src="https://unpkg.com/babel-core@5.8.38/browser.min.js"></script>
-    <script type="text/babel" src="bin/bundle.js"></script>
   </head>
   <body>
     <div id="content"></div>
+    <script type="text/javascript" src="./bin/bundle.js"></script>
```

Comment:
Here is the real doozie. By loading the script inside of the head, you're file was being run before the DOM was being loaded. So now that the script comes at the end of the body the page can load and create the DOM for your bundle to interact with. This is also why your DnD site was loading so slow. You don't want to load huge scripts in the head I don't think. Now it's faster by loading at the end and we killed the babel script since webpack handles it on the server